src/navigation/Routes.ios.js

```diff
@@ -157,6 +158,11 @@ function MainNavigator() {
         name={Routes.AVATAR_BUILDER}
         options={emojiPreset}
       />
+      <Stack.Screen
+        component={AvatarBuilder}
```

Comment:
can you explain why we have this route now duplicated? Can you make it a native sheet?

Reply:
This is because the positioning & avatar component is different on the wallet screen. Hopefully we can remove this once we convert the txn history screen back to React Native instead of a Swift view.